src/setupProxy.js

```diff
@@ -0,0 +1,12 @@
+const { createProxyMiddleware } = require('http-proxy-middleware')
+
+module.exports = function (app) {
+  app.use(
+    '/api',
+    createProxyMiddleware({
+      target: process.env.API_URL || 'http://backend:8080',
```

Comment:
How does that work? Isn't `process.env.API_URL` pointing to `:3000` / `:3001` again?